app/views/unattended/provisioning_templates/PXEGrub2/preseed_default_pxegrub2.erb

```diff
@@ -29,7 +29,7 @@ set default=0
 set timeout=<%= host_param('loader_timeout') || 10 %>
 
 menuentry '<%= template_name %>' {
-  linux<%= efi_suffix %>  <%= @kernel %> interface=auto url=<%= foreman_url('provision')%> ramdisk_size=10800 root=/dev/rd/0 rw auto BOOTIF=01-$net_default_mac <%= snippet("preseed_kernel_options").strip %>
```

Comment:
`net_default_mac` is a special value Grub2 supports, should we keep supporting this?

Reply:
we should change/remove it because we already have BOOTIF in the kernel-options-snippet and one BOOTIF should be sufficient